src/components/shared/misc/ImageUploader.vue

```diff
@@ -255,7 +281,9 @@
                         }))
 
                         tester.addEventListener('error', (() => {
-                            this.src = this.defaultSrc
+                            if (!this.defaultSrc) {
+                                this.src = ''
```

Comment:
Could we not pass through null though instead of a bool? This would break if we passed through true, right?

```js
this.src = this.defaultSrc || ''
```

Reply:
Good shout, done